cogent/examples/search/plat/linux/wordarray.ac

```diff
@@ -0,0 +1,62 @@
+/*
```

Comment:
Indentation in this file isn't consistent with the rest of the .ac files. I personally like this level of indentation (8 spaces) vs the (2 spaces) that we had earlier. But it shouldn't matter, as long as they are they same in each example.